chat/basics/ios.md

```diff
@@ -65,15 +58,11 @@ SKYContainer.default().chatExtension?.createConversation(
 })
 ```
 
-### Creating group chat by distinct participants *(distinctByParticipants)*
-
-By default, if you try to create conversations with same list of participants with `createConversation`. You will eventually create different conversations with the identical participants.
-
-This may or may not be a desire behavior in your application depends on your app design.
+### Creating group chat by distinct participants (`distinctByParticipants`)
 
-If you want to make sure each conversation contains distinct participants, you can set `distinctByParticipants` in options to be `true`.
+By default, `createConversation` allows you to create multiple conversations with identical participants. However, depending on application design, it may not be a desired behavior. Therefore, Skygear Chat provides `distinctByParticipants` option, which enables distinctive participants in every conversation.
 
-By default, `distinctByParticipants` is `false`.
+The default value of `distinctByParticipants` is `false`. If it is set to be `true`, then the participants cannot be identical to any existing conversation. Otherwise, an error will be returned.
```

Comment:
Just want to confirm if an error is really going to be returned? I remember the original version's behaviour is it will return the identical conversation if participants are identical.